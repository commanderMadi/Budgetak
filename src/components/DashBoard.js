import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const DashBoard = () => (
    <div>
        <h1>Dashboard</h1>
        <ExpenseList/>
        <ExpenseListFilters/>

    </div>
);

export default DashBoard;
