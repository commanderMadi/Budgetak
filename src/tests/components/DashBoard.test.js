import React from 'react';
import { shallow } from 'enzyme';
import  DashBoard from '../../components/DashBoard';

test('should render the dashboard page component', () => {
    const wrapper = shallow(<DashBoard />);

    expect(wrapper).toMatchSnapshot();
});