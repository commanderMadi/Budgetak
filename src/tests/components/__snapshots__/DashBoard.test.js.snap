// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the dashboard page component 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
  <Connect(ExpenseList) />
  <Connect(ExpenseListFilters) />
</div>
`;
